packages/fast-form-generator-react/src/form/form-item.select.style.ts

```diff
@@ -0,0 +1,70 @@
+import { Direction, ellipsis, localizeSpacing, toPx } from "@microsoft/fast-jss-utilities";
+
+function applypaddingStyle(): any {
+    return {
+        padding: localizeSpacing(Direction.ltr)(`${toPx(8)} ${toPx(24)} ${toPx(8)} ${toPx(8)}`)
```

Comment:
I think we'll want direction to be compiled at runtime instead of hard coded, correct?

Reply:
This was only done because of code climate issue - I can change back @janechu / @chrisdholt

Reply:
No - just talked with @janechu on this; can you create a task to pass this from a context so that this gets compiled correctly at runtime? No need to change at this point in time, but we'll want to go back and update.